cobc/parser.y

```diff
@@ -3529,6 +3530,51 @@ _prototype_procedure_division_header:
   }
 ;
 
+/* CONTROL DIVISION (GCOS extension) */
+
+_control_division:
+  /* empty */
+| CONTROL DIVISION TOK_DOT
+  {
+	  cb_verify (cb_control_division, "CONTROL DIVISION");
+  }
+  _default_section
+;
+
+_default_section:
+  /* empty */
+| DEFAULT SECTION TOK_DOT
+  _default_clauses
+  {
+	cobc_cs_check = 0;
+  }
+;
+
+_default_clauses:
+  /*empty*/
+| _default_accept_clause
+  _default_display_clause
+  TOK_DOT
+;
+
+_default_accept_clause:
+  /* empty */
+| ACCEPT _is word_or_terminal
+  {
+	  CB_PENDING (_("ACCEPT statement in DEFAULT SECTION"));
+	  /* (void) cb_build_accept_name ($3); */
```

Comment:
This one won't build anything but either change a flag in current_program or a global one.
Please also change to non-translated strings (dropping the `_()`, we don't want the translators to have to msgids for this quite rare feature)